index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dan's portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section id="skeleton">
+      <header>
+        <h1 id="welcome">Welcome</h1>
+        <a href="#" id="menu">
+          <img src="images/menu.svg" alt="Drop down menu" />
+        </a>
+      </header>
+      <div id="page">
+        <div id="intro">
+          <p id="denis">Hey there. I’m Denis</p>
+          <p id="trade">I’m a software developer</p>
+        </div>
+        <div id="advert">
+          <p>
+            I can help you build a product, feature or website. Look through
+            some of my work and experience! <br />
+            If you like what you see and have a project you need coded, <br />
+            don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="social">
+          <ul>
+            <li>
+              <a href="#"
+                ><img src="/images/social/twitter.svg" alt="Twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="/images/social/github.svg" alt="Github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="/images/social/linkedin.svg" alt="LinkedIn"
+              /></a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="/images/social/medium.svg" alt="Medium"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="/images/social/angellist.svg" alt="Arangellist"
+              /></a>
+            </li>
+          </ul>
+          <div id="union">
+            <img src="/images/union.png" alt="Union" />
+          </div>
```

Comment:
- [ ] Kindly make sure your drop-down icon works. There is an error in displaying.
Your design:
<img width="294" alt="Screenshot 2022-03-17 at 11 11 07 AM" src="https://user-images.githubusercontent.com/70269507/158765320-e35cef85-edcd-4327-a7cc-6516047b1069.png">

Figma design:
<img width="294" alt="Screenshot 2022-03-17 at 11 10 33 AM" src="https://user-images.githubusercontent.com/70269507/158765361-039629e3-3ddb-43fc-8e56-752198ac928f.png">